chaos_genius/alerts/utils.py

```diff
@@ -31,11 +32,19 @@ def webapp_url_prefix():
     return f"{CHAOSGENIUS_WEBAPP_URL}{forward_slash}"
 
 
-def save_anomaly_point_formatting(points: List[Dict]):
+def save_anomaly_point_formatting(points: List[Dict], scheduler_frequency: str = None):
     """Adds formatted fields to each point, to be used in alert templates."""
     for point in points:
         dt = datetime.datetime.strptime(point["data_datetime"], ALERT_DATETIME_FORMAT)
-        date = dt.strftime(ALERT_READABLE_DATETIME_FORMAT)
+
+        if scheduler_frequency is None:
+            dt_format = ALERT_READABLE_DATETIME_FORMAT
+        elif scheduler_frequency == "H":
+            dt_format = ALERT_READABLE_DATETIME_FORMAT
+        else:
+            dt_format = ALERT_READABLE_DATE_FORMAT
```

Comment:
Would be better to write it as this, in case other scheduler frequencies are added later:

```suggestion
        dt_format = ALERT_READABLE_DATETIME_FORMAT
        if scheduler_frequency is not None and scheduler_frequency == "D":
            dt_format = ALERT_READABLE_DATE_FORMAT
```